tools/lambda-promtail/lambda-promtail/kinesis.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"context"
+	"time"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/grafana/loki/pkg/logproto"
+	"github.com/prometheus/common/model"
+)
+
+func parseKinesisEvent(ctx context.Context, b *batch, ev *events.KinesisEvent) error {
+	for _, record := range ev.Records {
```

Comment:
`ev` is a pointer so it can be `nil` - so please check for this before accessing this field

Reply:
Thanks for a quick reply! I added a check for nil and added also a short test for the Kinesis part.